cli/dts/lib.deno.ns.d.ts

```diff
@@ -1418,9 +1418,9 @@ declare namespace Deno {
 
   export interface DirEntry {
     name: string;
-    isFile: boolean;
-    isDirectory: boolean;
-    isSymlink: boolean;
+    isFile: boolean | null;
+    isDirectory: boolean | null;
+    isSymlink: boolean | null;
```

Comment:
This is a breaking change

Reply:
Right; initially I was going to go with just having everything be false here and on second thought I think that makes alot more sense.

Deno.stat returns all as false without panicking.

```js
> Deno.statSync("/dev/fd/2")
{
  isFile: false,
  isDirectory: false,
  isSymlink: false,
  size: 0,
  mtime: 2021-02-22T14:55:17.914Z,
  atime: 2021-02-22T14:55:17.914Z,
  birthtime: 1970-01-01T00:00:00.000Z,
  dev: 608056912,
  ino: 649,
  mode: 8592,
  nlink: 1,
  uid: 501,
  gid: 4,
  rdev: 268435457,
  blksize: 131072,
  blocks: 0
}
```

Reply:
Went back to my first pass on this which is just returning false for all fields.